index.js

```diff
@@ -23,8 +23,10 @@
 
 // MetaModel handling
 const MetaModelUtil = require('./lib/metamodelutil');
+const DMCmodel = requires('./lib/dmcmodel')
```

Comment:
Can you update the reference and file names to DCSModel please? It'll be consistent with our docs.